components/automate-cli/pkg/verifyserver/services/batchcheckservice/batchcheckservice.go

```diff
@@ -51,7 +286,72 @@ func (ss *BatchCheckService) RunBastionCheck(check string, config models.Config,
 		resp[i].CheckType = check
 	}
 	resultChan <- resp
+}
+
+func (ss *BatchCheckService) GetPortsToOpenForCheck(check string) map[string]map[string][]int {
+	return ss.getCheckInstance(check).GetPortsForMockServer()
+}
+
+func (ss *BatchCheckService) getDeploymentState(config models.Config) (string, error) {
+	if config.Hardware.AutomateNodeCount > 0 {
+		for index, ip := range config.Hardware.AutomateNodeIps {
```

Comment:
if you are returning for all the cases below, then why does it need to be a For loop